src/app/components/card/card.component.html

```diff
@@ -0,0 +1,15 @@
+<mat-card class="c-card">
+    <img  class="c-card__image" src={{listing.images[0]}} alt="Card photo">
+    <mat-card-content class="c-card__content">
+      <h1 class="c-card__content--title">{{listing.title}}</h1>
+      <!-- <h4 class="c-card__content--description">{{listing.description}}</h4> -->
+      <h4 class="c-card__content--location">{{listing.location}}</h4>
+      <h2 class="c-card__content--price">{{listing.price}} lei</h2>
+    </mat-card-content>
+    <mat-card-actions class="c-card__actions">
+        <button mat-icon-button class="c-card__actions--button" (click)="setFavorite()">
+            <mat-icon *ngIf="favorite === false">favorite_border</mat-icon>
+            <mat-icon *ngIf="favorite === true" style="color: red;">favorite</mat-icon>
```

Comment:
To avoid writing duplicate code I suggest using a ternary conditional operator. 
This two lines 
` <mat-icon *ngIf="favorite === false">favorite_border</mat-icon>
   <mat-icon *ngIf="favorite === true" style="color: red;">favorite</mat-icon>`
   
   Could be written like this: 
   
  ` <mat-icon  [ngStyle]="{color: favorite? 'red' : 'white'}">  {{  favorite ? 'favorite ' : 'favorite_border'  }} </mat-icon>`